addons/mail/static/src/js/discuss.js

```diff
@@ -291,7 +291,7 @@ var Discuss = AbstractAction.extend({
         this._throttledUpdateThreads = _.throttle(
             this._updateThreads.bind(this), 100, { leading: false });
 
-        this.controlPanelModelConfig.modelName = 'mail.message';
+        this.actionModelConfig.modelName = 'mail.message';
```

Comment:
don't forget mail! See how to adapt
        - this.controlPanelModelConfig.modelName = 'mail.message';
        + this.actionModelConfig.modelName = 'mail.message';